Resources/public/js/privacycookie.js

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+(function (global, doc) {
+    var eZ = global.eZ = global.eZ || {};
+
+    /**
+     * Contains logic needed to display privacy banner.
+     *
+     * @class
+     * @param {Object} config
+     */
+    eZ.PrivacyCookieBanner = function (config) {
+        this.cookieName = config.cookieName || 'EzPrivacyCookieStatus';
+        this.bannerElement = document.getElementById(config.bannerId || 'privacy-cookie-banner');
```

Comment:
So, you only accept IDs. Why don't you accept other CSS selectors as well? Like `.cookie-banner`, `[class*="cookie-banner-]"`, etc (used CSS classname are jsut samples). You could use `document.querySelector` there.